python/paddle/fluid/layers/learning_rate_scheduler.py

```diff
@@ -307,6 +309,39 @@ def piecewise_decay(boundaries, values):
     return lr
 
 
+def cosine_decay(learning_rate, step_each_epoch, epochs):
+    """
+    Applies cosine decay to the learning rate.
+
+    when training a model, it is oftem recommended to lower the learning rate as the
+    training progresses. By using this function, the learning rate will be decayed by
+    following cosine decay strategy.
+    
```

Comment:
give the calculation formula for users here.

Reply:
ok